lib/suites.js

```diff
@@ -130,6 +136,16 @@ function _createEcdsaSd2023Suite({signer, options} = {}) {
   return diProof;
 }
 
+function _createEcdsaXi2023Suite({signer, options} = {}) {
+  // remove milliseconds precision
+  const date = new Date().toISOString().replace(/\.\d+Z$/, 'Z');
+  const extraInformation = options?.extraInformation || '';
```

Comment:
We can let the cryptosuite handle the default / validation of this -- and it no longer takes a string anyway:

```suggestion
  const extraInformation = options?.extraInformation;
```